src/pages/HomePage/Home.tsx

```diff
@@ -2,27 +2,24 @@ import "./home.scss";
 import React, { useState, useEffect } from "react";
 import "bootstrap/dist/css/bootstrap.min.css";
 import SliderComponent from "../../components/ui/slider/Slider";
+import Loader from "../../components/ui/loader/Loader";
 import { BreedType } from "../../types";
 
 function Home() {
-  const [breed, setInfo] = useState<BreedType[]>([]);
-  const [errorMessage, setError] = useState<string>("");
+  const [breed, setBreed] = useState<BreedType[]>([]);
```

Comment:
this variable should be called in plural form `breeds `instead of `breed` since it's array of items